test/600_proxy_docker_py_test.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "Run docker-py test suite against the proxy"
+
+assert_raises "docker_on $HOST1 ps | grep weaveproxy" 1
+weave_on $HOST1 launch-proxy
+assert_raises "docker_on $HOST1 ps | grep weaveproxy"
+docker_on $HOST1 run \
```

Comment:
Does this run command exit with a non-zero status if `integration_test.py` has test failures? It's not wrapped in an `assert_raises`, so it will cause `run_all.sh` to bail immediately with an `ABNORMAL TEST TERMINATION` message without running the remainder of the test suites...